### DISCUSSION TITLE
Trends page trending widget

### DISCUSSION THREAD

**Discussion by dnestico** [robertodoering/harpy#668] [Ideas]
I like the trending widget in the main Twitter app so a version of that in harpy would be nice, especially with the sports scores too as I'm a huge sports guy.

**EDIT: Just read that the API doesn't give access to personalized trends but I think this could still work for at least the trending image widget instead of the sports scores.**

<img width="600" alt="Screen Shot 2022-05-24 at 11 03 22 PM" src="https://user-images.githubusercontent.com/3034555/170171476-8346c950-ef30-4ac1-8eb9-7f46340d83a4.png">

**Comment by robertodoering** [comment]
This is the only information about trend that we get: https://developer.twitter.com/en/docs/twitter-api/v1/trends/trends-for-location/api-reference/get-trends-place

If Twitter add trends to their new api, they might add more information for us to display but as of right now it's not possible.
